operating/manage-vulnerabilities/vulnerability-management.adoc

```diff
@@ -14,9 +14,17 @@ include::modules/vulnerability-management-process.adoc[leveloffset=+1]
 
 include::modules/vulnerability-management-asset-assessment.adoc[leveloffset=+2]
 
-include::modules/vulnerability-management-view-applications-vulnerability.adoc[leveloffset=+3]
+[id="viewing-vulnerabilities"]
+== Viewing vulnerabilities
 
-include::modules/vulnerability-management-view-image-vulnerability.adoc[leveloffset=+3]
+{product-title-short} provides the following methods to view vulnerabilities discovered in your system:
+
+* To view application vulnerabilities by namespace or deployment, or to view vulnerabilities in an image, go to *Vulnerability Management (1.0)*.
+* To view vulnerabilities in applications running on clusters in your system, go to *Vulnerability Management (2.0)*. These vulnerabilities are referred to as _workload CVEs_. You can filter vulnerabilities by image, deployment, namespace, and cluster.
```

Comment:
I don't think that "go to" gives enough information here; I'd call out again that these are pages in a web portal or explain _how_ a user goes there.